model/task/task.go

```diff
@@ -1419,6 +1425,29 @@ func DeactivateStepbackTasksForProject(projectId, caller string) error {
 	return nil
 }
 
+// DeactivateStepbackTaskForProject deactivates and aborts the matching stepback task.
+// Will be used instead of LegacyDeactivateStepbackTasksForProject as part of EVG-17947
+func DeactivateStepbackTaskForProject(projectId, buildVariantName, taskName, caller string) error {
+	t, err := FindActivatedStepbackTaskByName(projectId, buildVariantName, taskName)
+	if err != nil {
+		return err
+	}
+	if t == nil {
+		return errors.New("no stepback task found")
```

Comment:
Should this error message include some info like the task name or whatnot so we can trace when this happens?